MyToolsYourToolsBackend/MyToolsYourToolsBackend.API/Controllers/RentController.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Cors;
+using Microsoft.AspNetCore.Mvc;
+using MyToolsYourToolsBackend.Application.Dtos;
+using MyToolsYourToolsBackend.Application.Services;
+
+// For more information on enabling Web API for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
+
+namespace MyToolsYourToolsBackend.API.Controllers
+{
+    [Route("api")]
+    [ApiController]
+    [EnableCors("MyToolsYourTools")]
+    public class RentsController : ControllerBase
+    {
+        private IRentService _rentService;
+        private IUserService _userService;
+
+        public RentsController(IRentService rentService, IUserService userService)
+        {
+            _rentService = rentService;
+            _userService = userService;
+        }
+
+        [HttpPost("rents")]
+        public IActionResult AddRent([FromBody]RentForCreationDto rentFromBody)
+        {
+            if (!_userService.CheckIfUserExists(rentFromBody.BorrowerId))
+            {
+                return NotFound();
+            }
+
+            int pointsRentCost = 100;
+
```

Comment:
To na pewno trzeba badzie przenieśc do serwisu. Ja to zrobie, bo przypisałem sobie juz te punkty

Reply:
ok